k9mail-library/src/main/java/com/fsck/k9/mail/internet/MimeMessage.java

```diff
@@ -156,6 +168,51 @@ public void setInternalSentDate(Date sentDate) {
     }
 
     @Override
+    public Date getResentDate() {
+        if (mResentDate == null) {
+            try {
+                DateTimeField field = (DateTimeField)DefaultFieldParser.parse("Resent-Date: "
+                        + MimeUtility.unfoldAndDecode(getFirstHeader("Resent-Date")));
+                mResentDate = field.getDate();
+            } catch (Exception e) {
```

Comment:
the function tries to get the Resent-Date header from the headers list.
if there aint one or an error happens, then the internal one will be returned.
it's a perfect clone of the existing function getSentDate()
`

     @Override
     public Date getSentDate() {
         if (mSentDate == null) {
             try {
                 DateTimeField field = (DateTimeField)DefaultFieldParser.parse("Date: "
                                       + MimeUtility.unfoldAndDecode(getFirstHeader("Date")));
                 mSentDate = field.getDate();
             } catch (Exception e) {
 
             }
         }
         return mSentDate;
     }
`